src/lib/Toast/Toast.svelte

```diff
@@ -0,0 +1,153 @@
+
+  properties = {
+    ...defaultToastProperties,
+    ...properties
+  };
+
+  function hideToast() {
+    showToast = false;
+  }
+
+  function handleAnimationEnd() {
+    dispatch('kill');
+  }
+
+  onMount(() => {
+    showToast = true;
+    timeout = setInterval(hideToast, properties.duration);
+  });
+
+  onDestroy(() => {
+    clearInterval(timeout);
+  });
+
+  function getToastType(type: string): ToastType | null {
+    switch (type) {
+      case 'success':
+        if (!leftIcon) {
+          leftIcon = 'https://sdk.breeze.in/gallery/icons/order-success-icon.svg';
+        }
+        return type;
+      case 'error':
+        if (!leftIcon) {
+          leftIcon = 'https://sdk.breeze.in/gallery/icons/error.svg';
+        }
+        return type;
+      case 'info':
+        if (!leftIcon) {
+          leftIcon = 'https://sdk.breeze.in/gallery/icons/info-white.svg';
+        }
+        return type;
+      case 'warn':
+        if (!leftIcon) {
+          leftIcon = 'https://sdk.breeze.in/gallery/icons/warning.svg';
+        }
+        return type;
+      default:
+        return null;
+    }
+  }
+</script>
+
+{#if showToast}
+  <div
+    class="toast {getToastType(properties.type)}"
+    style="border-radius: {properties.borderRadius};"
+    in:fly={{ x: 500, y: 0, duration: 400 }}
+    out:fly={{ x: 500, y: 0, duration: 800 }}
+    on:outroend={handleAnimationEnd}
+  >
+    <div class="toast-icon-container">
+      <div class="toast-icon-wrapper">
+        <img class="toast-icon" src={leftIcon} alt="toast-icon" />
+      </div>
+      <div style="color: {properties.textColor}">
+        {properties.message}
+      </div>
+    </div>
+
+    {#if properties.showRightIcon}
+      <div>
+        <div class="close-button" on:click={hideToast} on:keypress>
+          <img
+            class="toast-icon"
+            src="https://sdk.breeze.in/gallery/icons/close.svg"
+            alt="close-icon"
+          />
+        </div>
+      </div>
+    {/if}
+  </div>
+{/if}
+
+<style>
+  .toast {
+    padding: var(--toast-padding, 10px);
+    font-size: var(--toast-font-size, 14px);
+    height: var(--toast-height, auto);
+    width: var(--toast-width, auto);
+    align-items: var(--toast-align-items, center);
+    margin: var(--toast-margin, 0px 10px 10px 10px);
+    justify-content: space-between;
+    z-index: 1000;
+    display: flex;
+    position: absolute;
+    left: 0;
+    right: 0;
+  }
+
+  .toast-icon-container {
+    display: flex;
+    align-items: center;
+    gap: 10px;
+  }
+
+  .toast-icon-wrapper {
+    width: 20px;
+    height: 20px;
+  }
+
+  .toast-icon {
+    height: 100%;
+    border-radius: 50%;
+  }
+
+  .close-button {
+    width: var(--toast-width, 20px);
+    height: var(--toast-height, 20px);
+    cursor: var(--toast-cursor, pointer);
+    gap: 6px;
+    margin-left: 10px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .success {
+    background-color: #24aa5a;
```

Comment:
all styling must be exposed via css variables

Reply:
updated.